assets/containerisableComponents/marketingConsent/helpers.js

```diff
@@ -0,0 +1,54 @@
+// @flow
+
+// ----- Imports ----- //
+import { routes } from 'helpers/routes';
+import { logException } from 'helpers/logger';
+
+import type { Csrf as CsrfState } from 'helpers/csrf/csrfReducer';
+
+import { marketingConsentActionsFor } from './marketingConsentActions';
+
+// ----- Functions ----- //
+
+const requestData = (email: string, csrf: CsrfState) => ({
+  method: 'POST',
+  headers: { 'Content-Type': 'application/json', 'Csrf-Token': csrf.token || '' },
+  credentials: 'same-origin',
+  body: JSON.stringify({ email }),
+});
+
+// Fire and forget, as we don't want to interrupt the flow
+function sendMarketingPreferencesToIdentity(
+  optIn: boolean,
+  email: string,
+  dispatch: Function,
+  csrf: CsrfState,
+  scope: string,
+): void {
+
+  const setConsentApiError = marketingConsentActionsFor(scope).setAPIError;
+  const { setConfirmMarketingConsent } = marketingConsentActionsFor(scope);
```

Comment:
Would it be easier to do:

```jsx
const { setConsentApiError, setConfirmMarketingConsent } = marketingConsentActionsFor(scope);
```

and just rename it to `setConsentApiError` in the actions file?

Reply:
Good call, the final version will look like:
`const { setAPIError, setConfirmMarketingConsent } = marketingConsentActionsFor(scope);`